source/includes/mongodb-compatibility-table-rust.rst

```diff
@@ -5,47 +5,37 @@
    :class: compatibility-large
 
    * - Rust Driver Version
+     - MongoDB 5.2
      - MongoDB 5.1
      - MongoDB 5.0
      - MongoDB 4.4
      - MongoDB 4.2
      - MongoDB 4.0
      - MongoDB 3.6
-
    * - 2.1 [#2.1-limitation]_
-     -
-     - |checkmark|
-     - |checkmark|
-     - |checkmark|
-     - |checkmark|
-     - |checkmark|
-
-
+     - 
+     - 
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
    * - 2.0 [#limitations]_
-     -
-     - |checkmark|
-     - |checkmark|
-     - |checkmark|
-     - |checkmark|
-     - |checkmark|
-
+     - 
+     - 
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
    * - 1.1 [#limitations]_
-     -
-     -
-     - |checkmark|
-     - |checkmark|
-     - |checkmark|
-     - |checkmark|
-
-   * - 1.0 [#limitations]_
-     -
-     -
-     - |checkmark|
-     - |checkmark|
-     - |checkmark|
-     - |checkmark|
-
-The Rust driver is not compatible with MongoDB server versions older than 3.6.
```

Comment:
this is still true (perhaps you removed it intentionally but I see the Swift page still has a similar caveat)

Reply:
Did not mean to remove it, thanks for catching.